examples/react/start-mcp/src/routes/__root.tsx

```diff
@@ -22,7 +22,7 @@ export const Route = createRootRoute({
       },
     ],
   }),
-  component: RootComponent,
+  shellComponent: RootComponent,
```

Comment:
you would need to use `RootDocument` as the `shellComponent`
and then move `<TanStackRouterDevtools />` into `RootDocument`, then delete `RootComponent`

Reply:
Whoops sorry, I only looked at the linked line, I didn't notice more things were to be changed.

I copied from the start-basic example, is it ok now?

Reply:
yep!